content/en/docs/release-notes/release-notes-1.3.md

```diff
@@ -5,21 +5,27 @@ weight: 800
 type: "docs"
 ---
 
-This release prepares for the adoption of the upstream Kubernetes CSR API and improves interopability with Hashicorp Vault enterprise.
+This release prepares for the adoption of the upstream [Kubernetes CSR](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/) API and improves interoperability with HashiCorp Vault enterprise.
 A slew of bugs have also been squashed.
 
-Special thanks to the external contributors whose PRs landed in this release:
+Special thanks to the external contributors who contributed to this release:
 
 * [@teejaded](https://github.com/teejaded)
 * [@7opf](https://github.com/7opf)
 * [@yann-soubeyrand](https://github.com/yann-soubeyrand)
 * [@Kirill-Garbar](https://github.com/Kirill-Garbar)
 * [@joshuastern](https://github.com/joshuastern)
 * [@lalitadithya](https://github.com/lalitadithya)
+* [@johejo](https://github.com/lalitadithya)
+* [@alrs](https://github.com/alrs)
+* [@jsoref](https://github.com/jsoref)
+* [@RinkiyaKeDad](https://github.com/RinkiyaKeDad)
+* [@jonathansp](https://github.com/jonathansp)
+* [@OmairK](https://github.com/OmairK)
```

Comment:
Could we please also add @justinkillen https://github.com/justinkillen who helped a lot with the Multiple Certificate Requests issue?